### PR TITLE
ISPN-5932 Remove ClusterRegistry from query

### PR DESCRIPTION
**Pull request by anistor**
https://issues.jboss.org/browse/ISPN-5932

PARTIAL FIX, do not close the Jira yet. Security still uses ClusterRegistry.

### CODE REVIEW

File: core/src/main/java/org/infinispan/registry/ScopedKey.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 *
 * @author Mircea Markus
 * @since 6.0
+* @deprecated functionality replaced by {@link InternalCacheRegistry}
```
Comment (danberindei) [inline core/src/main/java/org/infinispan/registry/ScopedKey.java]: Please add a deprecation version and a `@Deprecated` annotation as well.

**Comment by anistor** [comment]
updated.

**Comment by anistor** [comment]
Rebased.

**Comment by tristantarrant** [comment]
Pushed, thanks @anistor

**Pull request closed without merging**